index.js

```diff
@@ -3,7 +3,41 @@
 module.exports = () => {
 
    return {
-      postcssPlugin: 'postcss-var-fallback',
+      postcssPlugin: 'postcss-css-var-fallback',
+
+      Root(root) {
+         root.walkDecls((decl) => {
+            // Capture declarations with include var()
+            const regex = new RegExp(/var\(/);
+
+            var getVarFallback = function(value) {
```

Comment:
Linting is failing on this but I am not sure the right way to refactor this. I am sure more of this file can be improved and I am eager to learn.

Reply:
Have a look at the linter output.

https://app.travis-ci.com/github/silvermine/postcss-css-var-fallback/jobs/537962832#L222

Our standards dictate that a variable definition not span multiple lines. A better approach might be to move the function to the returned object level:

```
return {
   myFunction: (value) => {
     // stuff
   },
   
   Root(root) {
     root.walk(() => {
      // stuff
      this.myFunction(arg);
    }
   }
}
```

Does that make sense?

Reply:
As discussed IRL we had to move the function into the `Root` hook due to object scope.

Reply:
ADIRL I couldn't move it outside but we improved things with your help